students/11th/daseulsong/main.html

```diff
@@ -0,0 +1,292 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>main-page</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <link href="style/common.css" rel="stylesheet" type="text/css" />
+    <link href="style/main.css" rel="stylesheet" type="text/css" />
+  </head>
+  <script
+    src="https://kit.fontawesome.com/f2f9d53b4c.js"
+    crossorigin="anonymous"
+  ></script>
+  <script src="js/main.js" defer></script>
+  <body>
+    <header>
+      <div class="header__container">
+        <ul class="logo">
+          <li><i class="fab fa-instagram"></i></li>
+          <li><p>|</p></li>
```

Comment:
```suggestion
          <li><span>|</span></li>
```
block 요소를 사용할 필요 없을 것 같습니다

Reply:
말씀하신 이런 불필요한 경우에  inline 요소가 아닌 block 요소를 사용했을 때 어떤 잠재적 문제가 있는지 궁금합니다.